### DISCUSSION TITLE
better dropdown, tooltip and popover. do we need to maintain compatibility to bootstrap.js/ boostrapVue?

### DISCUSSION THREAD

**Discussion by xvaara** [cdmoro/bootstrap-vue-3#355] [Ideas]
I've been thinking of rewriting the popperjs components without bootstrap.js, I did dropdown for a test previously. 

But there would be a better way: https://floating-vue.starpad.dev/

it has more features. For example nested dropdowns: https://floating-vue.starpad.dev/guide/component.html#sub-menu

But it would not be compatible with the all the current options (some might be made to work), and total get rid of popperOptions (which I'm sure isn't used that much).

@cdmoro @mariusa @aceofwings what to you think? I'd like some input before I but any work into this thing. What's the future goal of the project? Do we want to write out bootstrap.js or wait for improvements in it for ssr?

I can also create my own fork for ssr if you don't want these "radical" changes...

**Comment by aceofwings** [comment]
Hey @xvaara 

Right now there is no organized goal other then to build functional components and working with SSR compatibility in future releases.  In adding new features we look at adding at least several tests cases for to cover at least proper display and simple functions. I have never worked with any popper based elements so its kinda in your domain. At some point if we get time is to run through each component and see if their implementation is sufficient for a 1.0 release.
